packages/blockchain-wallet-v4-frontend/src/scenes/Home/Banners/RewardsBanner/index.tsx

```diff
@@ -0,0 +1,124 @@
+import React, { PureComponent } from 'react'
+import { FormattedMessage } from 'react-intl'
+import { connect, ConnectedProps } from 'react-redux'
+import { LinkContainer } from 'react-router-bootstrap'
+import { bindActionCreators } from '@reduxjs/toolkit'
+import styled from 'styled-components'
+
+import { Icon, Text } from 'blockchain-info-components'
+import { actions } from 'data'
+import { Analytics } from 'data/types'
+import { media } from 'services/styles'
+
+import { BannerButton, IconWrapper } from '../styles'
+
+const Wrapper = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  border: 1px solid ${(props) => props.theme.grey000};
+  border-radius: 8px;
+  padding: 20px;
+
+  ${media.atLeastLaptop`
+    height: 80px;
+    padding: 0 20px;
+  `}
+  ${media.mobile`
+    padding: 12px;
+    flex-direction: column;
+  `}
+`
+const Row = styled.div`
+  display: flex;
+  align-items: center;
+`
+const Column = styled.div`
+  display: flex;
+  flex-direction: column;
+
+  & > div:first-child {
+    margin-bottom: 4px;
+  }
+`
+const PendingIconWrapper = styled(IconWrapper)`
+  background-color: ${(props) => props.theme.blue100};
+`
+const Copy = styled(Text)`
+  display: flex;
+  align-items: center;
+  margin-right: 20px;
+  font-size: 12px;
+`
+
+class StartEarningRewards extends PureComponent<Props> {
+  constructor(props) {
+    super(props)
+    this.handleClick = this.handleClick.bind(this)
+  }
+
+  componentDidMount() {
+    this.props.analyticsActions.trackEvent({
+      key: Analytics.BANNER_REWARDS_VIEWED,
+      properties: {
+        device: 'WEB',
+        platform: 'WALLET'
+      }
+    })
+  }
+
+  handleClick() {
+    this.props.analyticsActions.trackEvent({
+      key: Analytics.BANNER_REWARDS_CLICKED,
+      properties: {
+        device: 'WEB',
+        platform: 'WALLET'
+      }
+    })
+  }
+
+  render() {
+    return (
+      <Wrapper>
+        <Row>
+          <PendingIconWrapper>
+            <Icon name='percentage' color='blue600' size='30px' />
+          </PendingIconWrapper>
+          <Column>
+            <Text size='20px' weight={600} color='grey800'>
+              <FormattedMessage
+                id='copy.banner_rewards_header'
```

Comment:
`copy. ...` is commonly used for text that are being used in several places like `copy.approve`
You may need to define something like 'scene.home.banners.reward.header'

Reply:
Didn't knew about that convention, is that in the notion docs?